package.json

```diff
@@ -85,6 +85,10 @@
     "testMatch": [
       "**/*.test.ts"
     ],
+    "coveragePathIgnorePatterns": [
+      "/node_modules/",
+      "tests/"
+    ],
```

Comment:
I tried configuring it to target all files in `src` instead using [`collectCoverageFrom`](https://jestjs.io/docs/en/configuration.html#collectcoveragefrom-array), but couldn't get it to work.